test/regress/regress0/bv/ackermann4.smt2

```diff
@@ -12,4 +12,5 @@
 (assert (= (f (f (f v0))) (g (f v0))))
 
 (check-sat)
+(get-model)
```

Comment:
The expected output of this method needs to be adjusted to include the model. Note that you want a test with a unique solution for this (because otherwise the test fails when CVC4 happens to pick a different model), so it would probably be better to create a separate test.

Reply:
Thanks! changed to regress0/print_lambda.cvc